src/main/java/Main.java

```diff
@@ -1,22 +1,30 @@
+import java.io.BufferedWriter;
 import java.io.File;
 import java.io.FileWriter;
 import java.io.IOException;
 
 public class Main {
 
     public static void main(String[] args) throws IOException {
-
+        Company company = new Company();
         File file = new File("employees.csv");
 
         if (file.exists()) {
             File stats = new File("stats.txt");
             stats.createNewFile();
+            company.readEmployees();
 
-//            FileWriter fileWriter = new FileWriter(stats);
-//            fileWriter.write("Średnia wypłata: 5000");
-//
-//            fileWriter.close();
-        }
+            FileWriter fileWriter = new FileWriter(stats);
+            BufferedWriter writer = new BufferedWriter(fileWriter);
+            writer.write("Średnia wypłata: " +  company.averageSalary(company.employees));
+            writer.newLine();
+            writer.write("Minimalna wypłata: " + company.lowestSalary(company.employees));
+            writer.newLine();
+            writer.write("Maksymalna wypłata: " + company.highestSalary(company.employees));
+            writer.newLine();
+            writer.write(company.employeeOfDepartment(company.employees));
 
+            writer.flush();
```

Comment:
Brakuje zamknięcia writera, przez co po zakończeni programu nadal może być "zajęty" plik.  Najlepiej do tego celu użyć try-with-resources.
Niepotrzebnie wywołujesz flush(). Jeśli będziesz zamykał plik, to flush wykona się automatycznie.